website/docs/sys-mgmt/backup-restore/restore.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Restore authentik
```

Comment:
I feel like the restore document is just a copy from the backup one. I would argue we can merge the two together.